### ISSUE TITLE
Scan replica sets fail to deploy

### ISSUE THREAD

**Issue by booboothefool** [openshift/compliance-operator#362]
```
example-scan-ip-10-0-163-83.ec2.internal-pod    2/2     Running            0          21m
example-scan-ip-10-0-171-206.ec2.internal-pod   1/2     Running            0          21m
example-scan-rs-5ddd5f8b-bvn26                  0/1     CrashLoopBackOff   
```
```
✗ oc logs example-scan-rs-5ddd5f8b-bvn26 
The command line argument 'scan-index' is mandatory.
```
(oc describe example-scan-rs-5ddd5f8b-bvn26  shows successful/no errors)
```
✗ oc version
Client Version: 4.4.5
Server Version: 4.3.10
Kubernetes Version: v1.16.2
```
```
apiVersion: compliance.openshift.io/v1alpha1
kind: ComplianceSuite
metadata:
  name: example
spec:
  autoApplyRemediations: false
  scans:
    - name: example-scan
      profile: xccdf_org.ssgproject.content_profile_coreos-ncp
      content: ssg-ocp4-ds.xml
      contentImage: quay.io/jhrozek/ocp4-openscap-content:latest
      nodeSelector:
        node-role.kubernetes.io/worker: ""
```
The example at https://github.com/openshift/compliance-operator#the-compliancesuite-object also gives the same error.

**Comment by jhrozek** [comment]
This sounds like a mixup of versions, how does the deployment of the operator look like?

**Comment by booboothefool** [comment]
@jhrozek 

```
apiVersion: apps/v1
kind: Deployment
metadata:
  name: compliance-operator
spec:
  replicas: 1
  selector:
    matchLabels:
      name: compliance-operator
  template:
    metadata:
      labels:
        name: compliance-operator
    spec:
      serviceAccountName: compliance-operator
      containers:
        - name: compliance-operator
          image: "quay.io/compliance-operator/compliance-operator:0.1.8"
          command:
          - compliance-operator
          - operator
          imagePullPolicy: Always
          env:
            - name: WATCH_NAMESPACE
              valueFrom:
                fieldRef:
                  fieldPath: metadata.namespace
            - name: POD_NAME
              valueFrom:
                fieldRef:
                  fieldPath: metadata.name
            - name: OPERATOR_NAME
              value: "compliance-operator"
            - name: OPENSCAP_IMAGE
              # Hardcoding this temporarily until its propagated to CI
              value: "quay.io/compliance-operator/openscap-ocp:1.3.3"
            - name: OPERATOR_IMAGE
              value: "quay.io/compliance-operator/compliance-operator:0.1.8"
            - name: DEFAULT_PROFILE_BUNDLES_IMAGE
              value: "quay.io/complianceascode/ocp4:latest"
      nodeSelector:
        node-role.kubernetes.io/master: ""
      tolerations:
        - key: "node-role.kubernetes.io/master"
          operator: "Exists"
          effect: "NoSchedule"
        - key: "node.kubernetes.io/unreachable"
          operator: "Exists"
          effect: "NoExecute"
          tolerationSeconds: 120
        - key: "node.kubernetes.io/not-ready"
          operator: "Exists"
          effect: "NoExecute"
          tolerationSeconds: 120
```

**Comment by booboothefool** [comment]
@jhrozek You are right sir.

I was using:
```
            - name: OPERATOR_IMAGE
              value: "quay.io/compliance-operator/compliance-operator:0.1.8"
            - name: DEFAULT_PROFILE_BUNDLES_IMAGE
              value: "quay.io/complianceascode/ocp4:latest"
```
because `"quay.io/compliance-operator/compliance-operator:latest"` wouldn't deploy the operator at all.

Had to downgrade the rest as well like this:
```
            - name: OPERATOR_IMAGE
              value: "quay.io/compliance-operator/compliance-operator:0.1.8"
            - name: LOG_COLLECTOR_IMAGE
              value: "quay.io/compliance-operator/resultscollector:0.1.8"
            - name: RESULT_SERVER_IMAGE
              value: "quay.io/compliance-operator/resultserver:0.1.8"
            - name: REMEDIATION_AGGREGATOR_IMAGE
              value: "quay.io/compliance-operator/remediation-aggregator:0.1.8"
```

**Comment by jhrozek** [comment]
I'm glad you found the issue. By the way, we released a new version yesterday, 0.1.10 that contains quite a significant number of changes.
